index.js

```diff
@@ -156,14 +174,15 @@ module.exports = function ({ types: t, template }) {
 
         const importArgNode = getImportArgPath(p).node
         const universalImport = getUniversalImport(p)
+        const existingChunkName = existingMagicCommentChunkName(importArgNode)
```

Comment:
I had to parse out the comment at the top so I don't accidentally catch the comment that gets added later.